README.md

```diff
@@ -88,6 +88,6 @@ Follow these steps to add documentation for a newly released version of etcd, vX
 [etcd.io]: https://etcd.io
 [Hugo]: https://gohugo.io
 [localhost:8888]: http://localhost:8888
-[LTS release]: https://nodejs.org/en/about/releases/
+[LTS release]: https://nodejs.org/en/download/
```

Comment:
Or it could be https://github.com/nodejs/release#release-schedule, which seems the most similar to that old page linked.